src/characters-transform.coffee

```diff
@@ -6,6 +6,9 @@ module.exports =
 class CharactersTransform
   initialize: (@source) ->
 
+  getNextRegions: ({sourceStartPosition}) ->
```

Comment:
Ah, I see you started on the plural version. Nice work. I think we should change it to just `getRegions` to match better with `getContent`.